src/system/Wizard/WizardStep.js

```diff
@@ -32,7 +42,11 @@ const WizardStep = React.forwardRef(
 		} else if ( active ) {
 			color = 'heading';
 		}
-
+		useLayoutEffect( () => {
+			if ( active && titleRef?.current && shouldFocusTitle ) {
```

Comment:
I think checking of `active` is enough and we can remove `shouldFocusTitle` param.

Reply:
@kat3samsin It would be active on the first load too, though, right? In this case we will forcefully focus on the first step when the user has yet to engage with the page. 

I would expect the user to prefer to navigate/read the page and _then_ visiting the first step. 

How would you handle this if you remove the shouldFocusTitle?

Reply:
I see what you mean, we should focus on the Wizard Header first rather than on the Wizard Step title on load.